cmd/snap/cmd_reboot.go

```diff
@@ -47,11 +47,11 @@ recovery system.
 When called without a system label and without a mode it will just
 trigger a regular reboot.
 
-When called without a system label but with a mode it will use the
-current system to enter the given mode.
+When called without a system label, the current system will be used for the
+"run" and "install" modes. The default recovery system will be used for the
```

Comment:
does it actually make sense to continue using current for install?

Reply:
Probably not, imo. I'll change it.